### ISSUE TITLE
Import https://www.geoportal.ch/services/wms/ktar in map.geo.admin.ch not possible

### ISSUE THREAD

**Issue by beat-github** [geoadmin/mf-geoadmin3#5222]
The import of the WMS https://www.geoportal.ch/services/wms/ktar in map.geo.admin.ch not possible.
Error message: "Fehler bei Hochladen".

![image](https://user-images.githubusercontent.com/6572456/106900077-60fb4500-66f6-11eb-9b1d-23e320fbc76a.png)

Expected result: WMS is shown.

Possibel relation to https://github.com/geoadmin/mf-geoadmin3/issues/5215

**Comment by procrastinatio** [comment]
Yes, this is related to https://github.com/geoadmin/mf-geoadmin3/issues/5229. But still, why move away from CORS?

**Comment by davidoesch** [comment]
Awaiting feedback / decision from @boecklic
